retriever/lib/datapackage.py

```diff
@@ -54,16 +54,16 @@ def get_replace_columns(dialect):
 
 
 def get_nulls(dialect):
-    """Get list of strings that denote null in the dataset"""
-    val = clean_input("nulls (separated by ';') (press return to skip): ",
+    """Get list of strings that denote missing value in the dataset"""
+    val = clean_input("missing-values (separated by ';') (press return to skip): ",
```

Comment:
That's OK with me. I'd recommended it be changed to it's current form to be consistent with 'table-url' etc. but it's just the user input request so I'm happy with whatever you (@henrykironde) think is best.